### PR TITLE
Migrate Customer block of Order view page

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Migrates Customer block of new Order view page.
| Type?         | refacto
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | #15821 
| How to test?  | :warning: Build assets before testing :warning: This PR does not add drop-down to these buttons (see https://prnt.sc/pk2y8a ), it will be done in another PR. Access `/admin-dev/index.php/sell/orders/orders/1/view` to see new Customer block.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15955)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/pages/order/view.js [hunk 1]
```diff
@@ -37,4 +43,35 @@ $(() => {
...
+  }
+
+  function handlePrivateNoteChange() {
+    const $submitBtn = $(OrderViewPageMap.privateNoteSubmitBtn);
+
+    $(OrderViewPageMap.privateNoteInput).on('input', (event) => {
+      const note = $(event.currentTarget).val();
+
+      if (note) {
+        $submitBtn.attr('disabled', false);
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/pages/order/view.js]: It's not an attribute, it's a property
```suggestion
        $submitBtn.prop('disabled', !note);
```

**Review by PierreRambaud — DISMISSED** [review]
One little thing

**Comment by sarjon** [comment]
@matks @PierreRambaud approve and QA?

**Comment by sarjon** [comment]
This PR does not add drop-down to these buttons (see https://prnt.sc/pk2y8a ), it will be done in another PR.

**Review by matks — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/pages/order/view.js [hunk 2]
```diff
@@ -37,4 +43,30 @@ $(() => {
...
+    const $icon = $btn.find('.material-icons');
+    $icon.text(isPrivateNoteOpened ? 'add' : 'remove');
+  }
+
+  function handlePrivateNoteChange() {
+    const $submitBtn = $(OrderViewPageMap.privateNoteSubmitBtn);
+
+    $(OrderViewPageMap.privateNoteInput).on('input', (event) => {
+      const note = $(event.currentTarget).val();
+      $submitBtn.attr('disabled', !note);
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/pages/order/view.js]: Use the property
```suggestion
      $submitBtn.prop('disabled', !note);
```

File: src/Core/Domain/Order/QueryResult/OrderCustomerForViewing.php [hunk 3]
```diff
@@ -163,4 +171,12 @@ public function getValidOrdersPlaced(): int
     {
         return $this->validOrdersPlaced;
     }
+
+    /**
+     * @return string
```
Comment (PierreRambaud) [inline src/Core/Domain/Order/QueryResult/OrderCustomerForViewing.php]:
```suggestion
     * @return string|null
```

**Comment by sarjon** [comment]
@matks can you re-approve? It was already finished (approved + qa) not it's not again. :sob:

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @sarjon

**Pull request merged**